packages/azure-http-specs/specs/azure/resource-manager/multi-service/client.tsp

```diff
@@ -0,0 +1,16 @@
+import "./service1.tsp";
+import "./service2.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+using Azure.ClientGenerator.Core;
+
+@client({
+  service: [
+    Azure.ResourceManager.MultiService.Compute,
```

Comment:
Can we add another case of selecting specific version of Compute/ComputeDisk?
I think this should be a supported scenario for compute service.